packages/mrml-core/src/mj_column/render.rs

```diff
@@ -8,6 +9,22 @@ struct MjColumnExtra<'a> {
 }
 
 impl<'root> Renderer<'root, MjColumn, MjColumnExtra<'root>> {
+    #[cfg(feature = "fragment")]
+    fn children_iter(&self) -> impl Iterator<Item = &MjBodyChild> {
+        fn folder<'root>(c: &'root MjBodyChild) -> Box<dyn Iterator<Item = &MjBodyChild> + 'root> {
```

Comment:
Not sure about that but we might not have to go through a Box.
Also, I'm not a big fan of having the function declared like this, just to use it once after.

Reply:
Perhaps a trait would be better? The problem I had with that is that the default impl in a trait can't access fields on a struct, so I have to reimplement the function anyway